pkg/rclone/driver.go

```diff
@@ -72,12 +90,27 @@ func NewControllerServer(d *Driver) *controllerServer {
 	}
 }
 
-func (d *Driver) Run() {
+func (d *Driver) Run() error {
 	s := csicommon.NewNonBlockingGRPCServer()
-	defer d.RcloneOps.Cleanup()
-	s.Start(d.endpoint,
+	s.Start(
+		d.endpoint,
 		csicommon.NewDefaultIdentityServer(d.csiDriver),
 		NewControllerServer(d),
-		NewNodeServer(d))
-	s.Wait()
+		NewNodeServer(d),
+	)
+	d.Server = s
+	go s.Wait()
+	err := d.RcloneOps.Run()
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
question: is this some go convention or can you just `return err` here instead of the `if` block?